docs/eBpfExtensions.md

```diff
@@ -296,14 +296,73 @@ The helper function ID for a general helper function must be in the range 0 - 65
 The parameter and return types for these helper functions must adhere to the `ebpf_argument_type_t` and
 `ebpf_return_type_t` enums.
 
-### 2.7 Registering Program Types and Attach Types
+### 2.7 Registering Program Types and Attach Types - eBPF Store
 The eBPF Execution Context loads an eBPF program from an ELF file that has program section(s) with section names. The
 prefix to these names determines the program type. For example, the section name `"xdp"` implies that the corresponding
-program type is `EBPF_PROGRAM_TYPE_XDP`. The Execution Context discovers the program type associated with a section
-prefix by reading the data from Windows registry. When an eBPF extension is installed, it must update the registry with
-the program types it implements along with the associated section prefixes.
-
-_Note: The registry location and data format are TBD. This is currently tracked by issue #223._
+program type is `EBPF_PROGRAM_TYPE_XDP`.
+
+The *Execution Context* discovers the program type associated with a section prefix by reading the data from Windows registry,
+in what is referred to aa the ***"eBPF Store"***.
```

Comment:
```suggestion
The *Execution Context* discovers the program type associated with a section prefix by reading the data from the ***"eBPF store"***, which is currently kept in the Windows registry.
```
Reword since it may not be in the registry in the future.